numba/core/withcontexts.py

```diff
@@ -311,6 +311,7 @@ def mutate_with_body(self, func_ir, blocks, blk_start, blk_end,
                                        func_globals=func_globals,
                                        func_closures=func_closures,
                                        )
+        typeanns["$cp"] = types.int32
```

Comment:
Could you please explain why `"$cp"` is added to the type annotations here, then removed a few lines further down, and then added back in again a few lines after that? Assuming there's a reason for this, should the other `$cp_*` related vars have the same?

Reply:
Looks like leftover from machete coding

Reply:
There should be just one `$cp` in the type annotation. The multi-exit fix adds the `$cp` variable that crosses the with-region. Post with-region, the `$cp` variable is used to determine the correct exit block.